### ISSUE TITLE
Enable profiling for database benchmarks

### ISSUE THREAD

**Issue by tuliren** [airbytehq/airbyte#8642]
- Depends on #8640.
- [x] Enable Sentry https://github.com/airbytehq/airbyte/issues/9328
- [ ] Enable Datadog
- [x] Make a decision on which one we want to support
- [ ] Next steps (if needed) for what platform changes are required to make it work (e.g: injecting env variables or something, do we want it in both OSS and Cloud, etc..)
- Relevant to #7956

**Comment by tuliren** [comment]
This issue depends on #8640, which depends on #8729. We probably cannot finish it this sprint.

**Comment by evantahler** [comment]
Closing in favor of https://github.com/airbytehq/airbyte/issues/15152
